prac_05/hex_colours.py

```diff
@@ -0,0 +1,30 @@
+"""
+CP1404/CP5632 Practical
+Colors in a dictionary
+"""
+
```

Comment:
This part is almost perfect, only if you can add some more comments or docstring to clarify your process, it would be perfect !